frontend/src/components/common/table/styles.module.scss

```diff
@@ -0,0 +1,32 @@
+.table {
+  width: 100%;
+  padding: 15px;
+  background: var(--background-gray-300);
+  border-radius: 10px;
+  border-spacing: 0;
+}
+
+.tableHeadRowHeader {
+  width: 52px;
+  height: 21px;
```

Comment:
Header cells should not be fixed - they may contain labels with width more than 50px + it's widths may be the same as relevant table body cells